Applications/MNIST/jni/main.cpp

```diff
@@ -270,6 +271,18 @@ int main(int argc, char *argv[]) {
     return 1;
   }
 
+  try {
+    auto optimizer = ml::train::createOptimizer("adam");
+    auto lr_scheduler = ml::train::createLearningRateScheduler("step");
+    lr_scheduler->setProperty({"learning_rate=0.0001, 0.00005"});
+    lr_scheduler->setProperty({"iteration=10"});
```

Comment:
It might be more clear how to use the step learning rate scheduler if more learning rate table is added.
```suggestion
    lr_scheduler->setProperty({"learning_rate=0.0001, 0.00009, 0.00007, 0.00005"});
    lr_scheduler->setProperty({"iteration=4, 6, 15"});
```

Reply:
fixed :)